src/actionCreators/index.js

```diff
@@ -19,31 +19,75 @@
 //
 
 import axios from 'axios';
-import { uploadStarted, uploadCompleted } from '../actions';
+import {
+  jobCreating,
+  jobCreated,
+  jobCreationFailed,
+  jobProcessing,
+  jobCompleted,
+  jobStatusCheckFailed,
+} from '../actions';
+import { API } from '../constants';
 
-export const createSigningJob = files => dispatch => {
-  console.log('** HERE **', files);
+const axi = axios.create({
+  baseURL: API.BASE_URL(),
+});
+
```

Comment:
FYI If you prefer using the name axios as a variable you can do:
import baseAxios from 'axios'; // or whatever you wanna name
const axios = baseAxios.create({
  baseURL: API.BASE_URL(),
});

Reply:
I'm OK with this. Thx.